scripts/messages/withdrawNativeSimple.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers } from 'hardhat';
+import { getPersonalWallet } from '../utils/wallet';
+
+async function main() {
+    const signer = getPersonalWallet(ethers.provider);
+    const bridgeAddress = process.env.BRIDGE_ADDRESS!;
+    const recipient = process.env.RECIPIENT!;
+    const amount = process.env.AMOUNT!;
+
+    if (!bridgeAddress || !recipient || !amount) {
+        throw new Error('BRIDGE_ADDRESS, RECIPIENT, and AMOUNT env vars required');
+    }
+
+    const bridge = await ethers.getContractAt('TestBridge', bridgeAddress, signer);
+
+    // Get the fee from the blockchain configuration
+    const nativeBridge = await bridge.nativeBridge();
+    const feeAmount = nativeBridge.config.fee;
+
+    console.log(`Using fee from blockchain: ${ethers.formatEther(feeAmount)} ETH`);
+
+    // Check sender balance
+    const senderAddress = await signer.getAddress();
+    const balance = await ethers.provider.getBalance(senderAddress);
+    console.log(`Sender address: ${senderAddress}`);
+    console.log(`Sender balance: ${ethers.formatEther(balance)} ETH`);
+
+    const withdrawalAmount = ethers.parseEther(amount);
```

Comment:
chore: check that the `withdrawalAmount` is divisible by `10^config.decimalScalingFactor`, i.e., throw an error if `withdrawalAmount % (10 ** nativeBridge.config.decimalScalingFactor) != 0`